Fabric/src/main/resources/fabric.mod.json

```diff
@@ -30,7 +30,7 @@
     "depends": {
       "fabricloader": ">=0.14",
       "fabric": "*",
-      "minecraft": "1.19.3",
+      "minecraft": "1.19.4",
```

Comment:
Is it really required that we need a seperate 1.19.4 version? Can't the current code just work by changing this to >=1.19.3 so that we can support both versions still?